gen-apidocs/generators/html.go

```diff
@@ -273,14 +275,16 @@ func (h *HTMLWriter) WriteDefinition(d *api.Definition) error {
 	h.writeAppearsIn(f, d)
 	h.writeFields(f, d)
 
-	// Definitions should not all show up individually in the TOC, as they are less interesting as jumping-in points to the docs.
-	// item := TOCItem{
-	// 	Level: 2,
-	// 	Title: nvg,
-	// 	Link:  linkID,
-	// 	File:  fn,
-	// }
-	// h.currentTOCItem.SubSections = append(h.currentTOCItem.SubSections, &item)
+	// Definitions are added to the TOC to enable the generator to later collect
+	// all the individual definition files, but definitions will not show up
+	// in the nav treet because it would take up too much screen estate.
```

Comment:
I still believe the definitions can/should be listed in the navigation sidebar because Kubernetes API is unlike other APIs, as a declarative API, it is built around various definitions rather than operations.

Reply:
I was going based on the old docs, which did not show the definitions in the TOC. We can bring them back in this PR, but should they be hidden with CSS? Otherwise the TOC might still be needlessly long?

Reply:
The list of definitions doesn't have to be shown in the TOC, but I think we need a way for users to quickly find 'DeploymentSpec', for example.

Reply:
But if it's hidden, how will they find it?

Reply:
I think we can continue collapse the list of definitions.
Yes we have a lot of defintions, and that is something we have to live with.
Or in the future, we can split the generated HTML into several files,
simplify the navigator, etc.

Reply:
OK I brought the definition items in the TOC back and will add CSS in the website PR to permanently hide them again.